rho/utilities.py

```diff
@@ -118,6 +125,48 @@
     + CONNECTION_FACTS_TUPLE
 
 
+def threaded_tailing(path, ansible_verbosity=0):
+    """Follow and provide output using a thread
+    :param path: path to file to follow
+    :param ansible_verbosity: the verbosity level
+    """
+
+    _thread.start_new_thread(tail_and_follow, (path, ansible_verbosity))
+
+
+def tail_and_follow(path, ansible_verbosity):
+    """Follow and provide output
+    :param path: tuple containing thepath to file to follow
+    :param ansible_verbosity: the verbosity level
+    """
+    if len(path) > 0:  # pylint: disable=len-as-condition
+        truncate = 1
+        if ansible_verbosity > 0:
+            truncate = ansible_verbosity
+
+        print_line = truncate
+        truncated = False
+
+        # pylint: disable=no-member
+        for line in sh.tail('-f', '-n', '+0', path, _iter=True):
+            line = line.strip('\n')
+            if line.startswith('TASK') or line.startswith('PLAY'):
+                print(line)
+                print_line = truncate
+                truncated = False
+            elif print_line > 0:
+                line_len = len(line)
+                char_truncate = truncate * 100
+                if line_len > char_truncate:
+                    print(line[0:char_truncate])
```

Comment:
Can we print ` ...` or something similar at the end of truncated lines to make it more clear what's happening?